components/post_view/post_body/post_body.tsx

```diff
@@ -222,8 +222,11 @@ export default class PostBody extends React.PureComponent<Props, State> {
                     {messageWithAdditionalContent}
                     {fileAttachmentHolder}
                     <div className='post__body-reactions-acks'>
-                        {this.props.isPostAcknowledgementsEnabled && post.metadata?.priority?.requested_ack && (
-                            <PostAcknowledgements postId={post.id}/>
+                        {this.props.post.state !== Posts.POST_DELETED && this.props.isPostAcknowledgementsEnabled && post.metadata?.priority?.requested_ack && (
```

Comment:
`post` is already destructured from `this.props`, so you can just use it here as well